new-features/index.html

```diff
@@ -110,56 +110,39 @@
         </header>
         <div id="main" class="container cf">
             <section>
-                <h1 id="new">New GenePattern Notebook Features June 2019</h1>
+                <h1 id="new">New GenePattern Notebook Features<br>
+                October 2019</h1>
                 <hr>
                 <p>The following new features have recently been added to the GenePattern Notebook environment.</p>
 
                 <h2><a href="/basic-features/">General Features</a></h2>
                 <ul>
-                    <li>Table of contents functionality has been added to the GenePattern Notebook environment. To access
-                        this functionality, click the new <em>Table of Contents</em> button in the toolbar. This will open a
-                        panel on the left-hand side of the screen. From this panel, various
-                        options can be selected, including the automatic numbering of headers and the generation of
-                        automatically-updated table of contents cells.</li>
-                    <li>Public notebooks now have improved preview functionality. To access a notebook's preview, click
-                        any public notebook on the <em>Notebook Library</em> tab and select the <em>Preview</em> button.
-                        This will bring you to a new page with basic information about the notebook and a complete
-                        preview image. From here you can download the notebook or run a copy of the notebook. You may
-                        also use the URL of the preview page as a sharable link which can be distributed to collaborators.</li>
-                    <li>Users now have the ability to add comments to public notebooks. This can be done from either the
-                        notebook via the <em>Comments</em> button on the toolbar, or from the <em>Notebook Library</em> tab.</li>
-                    <li>Users may now search over all public notebooks at once in the <em>Notebook Library</em> tab.</li>
-                    <li>New users are now presented with a tour of GenePattern Notebook features when they first log in.
-                        Existing users can view this tour from the help panel, accessed by clicking on the <em>i</em>
-                        icon in the lower right.</li>
+                    <li><a href="/programmatic/#uibuilder">UI Builder</a> cells now support file uploads larger than 2GB.</li>
```

Comment:
The link <a href="/programmatic/#uibuilder"> is not working for me.

1- Should we use an absolute path to the URL?
2- Should it be #ui-builder?

Reply:
It will only work when it's live

Reply:
I just tested it and I don't understand why #ui-builder is *not* the correct syntax (I thought spaces were replaced by dashes). Oh well. This is correct then. I'll hold off on accepting the PR until we confirm wether or not we need the `..` in the <a> tags.